cmds/webboot/webboot.go

```diff
@@ -108,3 +149,50 @@ func download(URL, fPath string) error {
 	}
 	return nil
 }
+
+// will call a function to get all the block devices, then mount it one by one
```

Comment:
For consistency with your other good comments, :
// getCachedIsos will call a function ...

Reply:
sorry it is not what getCachedIsos will do. it's just a todo to remind me what to do next. I will mark them by "todo:" . sorry